index.js

```diff
@@ -44,7 +44,7 @@ types[VISA] = {
   prefixPattern: /^4$/,
   exactPattern: /^4\d*$/,
   gaps: [4, 8, 12],
```

Comment:
We got the number 18 from this article: https://www.nccgroup.trust/us/about-us/newsroom-and-events/blog/2016/november/prepare-for-19-digit-credit-cards/